sequencer/database/migrations/0001.sql

```diff
@@ -26,8 +26,6 @@ CREATE TABLE batch (
     batch_num BIGINT UNIQUE NOT NULL,
     eth_block_num BIGINT NOT NULL REFERENCES block (eth_block_num) ON DELETE CASCADE,
     forger_addr BYTEA NOT NULL, -- fake foreign key for coordinator
-    fees_collected BYTEA NOT NULL,
-    fee_idxs_coordinator BYTEA NOT NULL,
```

Comment:
a minor suggestion to comment these out instead of deleting for consistency (i think we've been commenting out everything irrelevant so far)